src/components/logout-button.tsx

```diff
@@ -0,0 +1,19 @@
+import { getServerAuthSession } from "~/server/auth";
+
+export default async function LogoutButton() {
+  const session = await getServerAuthSession();
+
+  if (session) {
+    return (
+      <a
+        href="/api/auth/signout"
```

Comment:
Similar to `login-button.tsx`, we can simply return the component itself instead of checking for whether the user is logged in at this level. Additionally, we can also call the `signOut()` function as follows:

```javascript
"use client";

import { Button } from "./ui/button";
import { signOut } from "next-auth/react";

export default function LogoutButton() {
  return (
    <Button
      onClick={() => {
        signOut({ callbackUrl: "/" });
      }}
      type="button"
      className="rounded-lg bg-red-700 px-5 py-2.5 text-sm font-medium text-white hover:bg-red-800 focus:outline-none focus:ring-4 focus:ring-red-300 dark:bg-red-600 dark:hover:bg-red-700 dark:focus:ring-red-900"
    >
      Sign Out
    </Button>
  );
}

```